src/components/TestRunList.tsx

```diff
@@ -56,8 +59,61 @@ const TestRunList: React.FunctionComponent<{
     setDeleteDialogOpen(!deleteDialogOpen);
   };
 
+  const doesHaveUnapprovedImage = (): boolean => {
+    let status = false;
+    items.forEach((eachItem) => {
```

Comment:
could you replace `forEach` with `some` ?
https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/some